src/index.js

```diff
@@ -1153,28 +1153,35 @@ const initialize = async () => {
       console.log(
         `Contract mined! address: ${hstContract.address} transactionHash: ${hstContract.deployTransaction.hash}`,
       );
-      tokenAddress.innerHTML = hstContract.address;
-      watchAsset.disabled = false;
+      tokenAddresses.innerHTML += `${hstContract.address} `;
+      watchAssets.disabled = false;
       transferTokens.disabled = false;
       approveTokens.disabled = false;
       transferTokensWithoutGas.disabled = false;
       approveTokensWithoutGas.disabled = false;
     };
 
-    watchAsset.onclick = async () => {
-      const result = await ethereum.request({
-        method: 'wallet_watchAsset',
-        params: {
-          type: 'ERC20',
-          options: {
-            address: hstContract.address,
-            symbol: tokenSymbol,
-            decimals: decimalUnitsInput.value,
-            image: 'https://metamask.github.io/test-dapp/metamask-fox.svg',
+    watchAssets.onclick = async () => {
+      const contractAddresses = tokenAddresses.innerHTML.trim().split(' ');
```

Comment:
maybe we can make this comma delimited like the other fields. i don't think we need the trim()

Reply:
Would that change look like on L1156 to be
```tokenAddresses.innerHTML += `${hstContract.address},`;```
and then remove the `.trim()` part here?